LoopKitUI/CGMManagerUI.swift

```diff
@@ -63,28 +66,21 @@ extension CGMManagerUI {
     }
 }
 
-public protocol CGMManagerCreateDelegate: AnyObject {
-    /// Informs the delegate that the specified cgm manager was created.
+public protocol CGMManagerOnboardingDelegate: AnyObject {
+    /// Informs the delegate that the specified CGM manager was created.
     ///
     /// - Parameters:
-    ///     - cgmManager: The cgm manager created.
-    func cgmManagerCreateNotifying(didCreateCGMManager cgmManager: CGMManagerUI)
-}
-
-public protocol CGMManagerCreateNotifying {
-    /// Delegate to notify about cgm manager creation.
-    var cgmManagerCreateDelegate: CGMManagerCreateDelegate? { get set }
-}
+    ///     - cgmManager: The CGM manager created.
+    func cgmManagerOnboarding(didCreateCGMManager cgmManager: CGMManagerUI)
 
-public protocol CGMManagerOnboardDelegate: AnyObject {
-    /// Informs the delegate that the specified cgm manager was onboarded.
+    /// Informs the delegate that the specified CGM manager was onboarded.
     ///
     /// - Parameters:
-    ///     - cgmManager: The cgm manager onboarded.
-    func cgmManagerOnboardNotifying(didOnboardCGMManager cgmManager: CGMManagerUI)
+    ///     - cgmManager: The CGM manager onboarded.
```

Comment:
[nit] should `CGM` be used consistently (e.g., see `var cgmManagerCreateDelegate: CGMManagerCreateDelegate? { get set }` above)?

Reply:
Not sure I understand. It is being used consistently, at least in my mind. "CGM" is an abbreviation and therefore, by default, should be capitalized (thus, "The CGM manager onboarded."). However, Swift coding standards override that default when it is used as the first part of a property or argument name (thus, "cgmManager").

Reply:
Did I miss something?